### PR TITLE
Resolve comments & restructure vc_di models

### PR DESCRIPTION
**Pull request by sarthakvijayvergiya**

### CODE REVIEW

File: aries_cloudagent/indy/models/cred_abstract.py [hunk 1]
```diff
@@ -153,200 +152,3 @@ class Meta:
...
-        required=True,
-        validate=NUM_STR_WHOLE_VALIDATE,
-        metadata={
-            "description": "Nonce in credential abstract",
-            "example": NUM_STR_WHOLE_EXAMPLE,
-        },
-    )
-
-
-class BindingMethod(BaseModel):
```
Comment (gvelez17) [inline aries_cloudagent/indy/models/cred_abstract.py:259]: huh no one else uses this we are sure?

**Review by gvelez17 — COMMENTED** [review]
Looks good to me, just do we know if the changes to existing functions are all under test coverage?  in python can be less than safe to make changes if no test coverage since there is no build step to save us - we are confident the deleted functions are unused?

File: aries_cloudagent/indy/models/cred_abstract.py (hunk 1 above)
Comment (sarthakvijayvergiya) [reply aries_cloudagent/indy/models/cred_abstract.py:259]: Yes, it has just moved to a new location. And used only in vc_di.

File: aries_cloudagent/anoncreds/holder.py [hunk 2]
```diff
@@ -316,6 +316,7 @@ async def store_credential_w3c(
                 credential_definition,
                 rev_reg_def,
             )
+            # TODO
```
[the remark this answers is not in the document]
Comment (ianco) [reply aries_cloudagent/anoncreds/holder.py:319]: Agree should say `# TODO ... <what is the work required>`

File: aries_cloudagent/protocols/issue_credential/v2_0/formats/vc_di/handler.py [hunk 3]
```diff
@@ -304,6 +328,7 @@ async def create_request(
         ]
 
         ledger = self.profile.inject(BaseLedger)
+        # TODO
```
Comment (ianco) [inline aries_cloudagent/protocols/issue_credential/v2_0/formats/vc_di/handler.py:331]: Same comment as above re `TODO`

**Review by ianco — APPROVED** [review]
Looks pretty good.  Just the comment about the `TODO` comments.  There are a couple other things I want to look at in more detail but please merge this to get all the changes into the aca-py PR so I can review there ...
Thanks!

**Comment by sarthakvijayvergiya** [comment]
Thanks, I will merge it and update todo comments shortly!

**Pull request merged**